framework/include/tinyalsa/tinyalsa.h

```diff
@@ -262,6 +262,24 @@ struct pcm;
 struct pcm *pcm_open(unsigned int card, unsigned int device, unsigned int flags, const struct pcm_config *config);
 
 /**
+* @brief Stops a PCM. Any data present in the buffers will be dropped.
+*
+* @param[in] A PCM handle.
+* @returns On success, zero; on failure, a negative number.
+* @since Tizen RT v1.1
```

Comment:
1. Please mark it as 1.x, because 1.1 was released already. We should modify it later when new version is confirmed.
2. If you modify public APIs, you should process ACR first.

Reply:
Done